packages/commands/package.json

```diff
@@ -0,0 +1,45 @@
+{
+	"name": "@wordpress/commands",
+	"version": "0.1.0",
+	"description": "Handles the commands menu.",
+	"author": "The WordPress Contributors",
+	"license": "GPL-2.0-or-later",
+	"keywords": [
+		"wordpress",
+		"gutenberg",
+		"commands",
+		"k"
+	],
+	"homepage": "https://github.com/WordPress/gutenberg/tree/HEAD/packages/commands/README.md",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/WordPress/gutenberg.git",
+		"directory": "packages/commands"
+	},
+	"bugs": {
+		"url": "https://github.com/WordPress/gutenberg/issues"
+	},
+	"engines": {
+		"node": ">=12"
```

Comment:
Where does this come from? It's out of sync with other directives, like `.nvmrc`, which declare version 14.

In general, do we need to duplicate these declarations across packages?

Reply:
I copied other packages. Maybe that changed after the PR was started?